com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/JavaDebugDelegateCommandHandler.java

```diff
@@ -50,6 +52,8 @@ public Object executeCommand(String commandId, List<Object> arguments, IProgress
             return UsageDataStore.getInstance().fetchAll();
         } else if (UPDATE_DEBUG_SETTINGS.equals(commandId)) {
             return DebugSettingUtils.configDebugSettings(arguments);
+        } else if (VALIDATE_LAUNCHCONFIG.equals(commandId)) {
+            return new ResolveMainClassHandler().validateLaunchConfig(arguments);
```

Comment:
How about make class ResolveMainClassHandler a utility class(with a changed name) and not new ResolveMainClassHandler every time